samples/hello-connector-solace/src/main/java/com/solace/quarkus/samples/HelloConsumer.java

```diff
@@ -11,61 +11,56 @@
 import com.solace.quarkus.messaging.outgoing.SolaceOutboundMetadata;
 
 import io.quarkus.logging.Log;
+import io.smallrye.mutiny.Multi;
 
 @ApplicationScoped
 public class HelloConsumer {
+
     /**
-     * Publish a simple message using TryMe in Solace broker and you should see the message published to topic
+     * Publishes message to topic hello/foobar which is subscribed by queue.foobar
+     *
+     * @see #consumeMessage(SolaceInboundMessage)
+     * @return
+     */
+    @Outgoing("hello-out")
```

Comment:
Earlier we are creating a outbound message with ack and nack callbacks which used to trigger acks to broker. Since we removed it i believe it is not calling the ack method which sends ack to broker

Reply:
This comment mentions the outgoing message, unless you are chaining from an incoming message you don't need to specify ack/nacks for outgoing messages.
For incoming messages, in most cases, ack/nack is already managed by the framework.